projects/stargazer/plugins/other/radius/radius.h

```diff
@@ -24,7 +26,10 @@ namespace STG
 
         private:
             mutable std::string errorStr;
+            std::jthread m_thread;
+            std::mutex m_mutex;
             bool isRunning;
```

Comment:
I know that the old plugins use `isRunning` and `errorStr`, but in the new code let's stick to the `m_` naming scheme for the private members. So please change them to `m_running` and `m_errorStr`.